src/main/java/seedu/address/logic/parser/clisyntax/CliSyntax.java

```diff
@@ -1,4 +1,6 @@
-package seedu.address.logic.parser;
+package seedu.address.logic.parser.clisyntax;
+
```

Comment:
I am wondering if since the Internship has been categorized into InternshipCliSyntax, perhaps this could be also categorized into ItemCliSyntax, or were u planning to convert this class into that in the future?

Reply:
I didn't change this yet since this was the original file but I suppose like it can be changed now too.